src/components/Steps.tsx

```diff
@@ -47,7 +53,8 @@ const Steps = () => {
                 hint: 'Confirm this transaction in your wallet',
                 status: 'loading',
             })
-            const txResponse = await signer.sendTransaction(txData)
+            const tx = await handlePreTxGasEstimate(signer, txData)
```

Comment:
Is there anywhere else this is missing?

Reply:
Ah I forgot to mention this, but basically I didn't notice it causing a failure but I noticed when comparing code that it wasn't here and everywhere else I looked we had it so I added it in. The only other places that doesn't have it is handleWrapEther (which I wrote...so I'll make a note to add it in there as well)